drivers/rtc/Kconfig.stm32_rtc

```diff
@@ -15,7 +15,7 @@ config RTC_STM32
 	select USE_STM32_LL_EXTI
 	select NEWLIB_LIBC
```

Comment:
While not specific to your change, but does the RTC driver really need NEWLIB?

Reply:
It helps with the conversion between calendar daytime and UNIX timestamp.